docs/guides/authentication.md

```diff
@@ -1,15 +1,30 @@
 ---
-sidebar_position: 1
+sidebar_position: 2
 tags:
   - authentication
+  - non-custodial
+  - custodial
   - sign-in
   - sign-up
   - login
-  - non-custodial
 ---
 
 # Authentication
 
+Depending on the use case and endpoint either API key or Sign-in-with-ethereum (SIWE) authentication may be used.
+Our SIWE method allows client applications to act on behalf of users within scopes explicitly granted by the user0..
```

Comment:
is "user0" a typo?

Reply:
Haha. I know how that happened my sliding keyboard scraping on the underside of my desk.